src/pymodaq_plugins_newport/daq_move_plugins/daq_move_Newport_XPS_Q8.py

```diff
@@ -0,0 +1,294 @@
+from pymodaq.control_modules.move_utility_classes import DAQ_Move_base, comon_parameters_fun, main, DataActuatorType,\
+    DataActuator  # common set of parameters for all actuators
+from pymodaq.utils.daq_utils import ThreadCommand # object used to send info back to the main thread
+from pymodaq.utils.parameter import Parameter
+from qtpy.QtCore import QThread
+from pymodaq_plugins_newport.hardware import XPS_Q8_drivers 
+import sys
+
+from time import perf_counter_ns
+
+class XPSPythonWrapper():
+    """Simplified XPS wrapper, calls methods from the wrapper given by Newport. See XPS_Q8_drivers"""
+    
+    def __init__(self, ip:str = None, port:int = None, group:str = None, positionner:str = None, plugin = None):
+        #init the wrapper given by Newport and some attributes
+        self.myxps = XPS_Q8_drivers.XPS()   #Instanciate the driver from Newport
+        
+        #keep a ref of the plugin to emit (error) messages
+        self._plugin = plugin 
```

Comment:
your driver should ingeneral be unaware of your plugin as in a nice world some other people could use it for another task in another program. if you want to pass errors, use logger:
```
from pymodaq.utils.logger import set_logger, get_module_name
logger = set_logger(get_module_name(__file__))

logger.error('my error')
```
or you can raise an error and catch it in your plugin to transform it into a emit_status

Reply:
done on commit [abcc736](https://github.com/PyMoDAQ/pymodaq_plugins_newport/pull/6/commits/abcc73647d8281f5c30ddc2ed09147e4a6cc00bd). I chose to raise a custom "XPSError" to keep the hardware wrapper independant from pymodaq, and to catch those in the plugin. Thanks for the suggestion